hrm-domain/hrm-service/src/profile/profile-data-access.ts

```diff
@@ -78,16 +78,28 @@ export const getIdentifierWithProfiles =
     identifier,
     identifierId,
   }: IdentifierParams): Promise<TResult<IdentifierWithProfiles | null>> => {
+    const params = { accountSid, identifier, identifierId };
     try {
-      const result = await txIfNotInOne<{ data: IdentifierWithProfiles }>(task, async t =>
-        t.oneOrNone(joinProfilesIdentifiersSql, {
-          accountSid,
-          identifier,
-          identifierId,
-        }),
-      );
+      const data = await txIfNotInOne<IdentifierWithProfiles>(task, async t => {
```

Comment:
Perhaps a short comment to explain that this is 2 queries for performance reasons, to stop the next well meaning refactorer coming along and recreating the bug :-)